File: repayment-server/src/main/java/com/example/repayment_server/client/dto/EntryResponseDto.java
package com.example.repayment_server.client.dto;

import lombok.*;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@NoArgsConstructor
@AllArgsConstructor
@Builder
@Getter
@Setter
public class EntryResponseDto {

    private Long entryId;

    private Long applicationId;

    private BigDecimal entryAmount;

    private LocalDateTime createdAt;

    private String createdBy;

    private LocalDateTime updatedAt;

    private String updatedBy;
}

File: ms-loan-frontend/src/app/dtos/entry-response-dto.ts
import { CommunicationStatus } from './communitation-status';

export interface EntryResponseDto {
  entryId: number;
  applicationId: number;
  entryAmount: number;
  communicationStatus: CommunicationStatus;
  createdAt: string;
  createdBy: string;
  updatedAt: string;
  updatedBy: string;
}
